docs/join-waitlist-form.ftd

```diff
@@ -24,9 +24,6 @@ sections: $lib.sitemap.sections
 
 -- ds.page.body:
 
-We have developed the `join-waitlist-form` component for [admint.io](https://admint.io/), 
-which matches the design currently displayed on the live site.
-
 -- ds.h3: How to use?
```

Comment:
If the suggestion of `h1` is correct, please make appropriate change everywhere.